configs/grumphp-extension.yml

```diff
@@ -37,75 +38,84 @@ grumphp:
         - "\\$_[A-Z_]+\\["
       regexp_type: E
       triggered_by:
-        - php
         - inc
         - install
         - module
-        - theme
+        - php
         - profile
+        - theme
     git_branch_name:
       whitelist:
-        - "#^\\d+\\(.\\d+)?\\.x|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
+        - "#^\\d+(.\\d+)?\\.x|master|develop|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
     git_commit_message:
-      enforce_no_subject_trailing_period: false
-      max_body_width: 80
-      max_subject_width: 80
```

Comment:
Op welke Drupal standaard baseer je deze wijzigingen?

Reply:
See https://chris.beams.io/posts/git-commit/

The default of GrumPHP are based on these recommendations: https://chris.beams.io/posts/git-commit/#limit-50
And GitHub will by default truncate the message when it is longer then 50 chars...